src/com/owncloud/android/files/services/FileDownloader.java

```diff
@@ -80,6 +86,10 @@
     public static final String EXTRA_LINKED_TO_PATH = "LINKED_TO";
     public static final String ACCOUNT_NAME = "ACCOUNT_NAME";
 
+    public static final String KEY_FILE_REMOTE_PATH = "FILE_REMOTE_PATH";
+    public static final String KEY_ACCOUNT_NAME = "ACCOUNT_NAME";
```

Comment:
The constants are public, so they might be used in many places.

But, who should be the responsible of its declaration? Who should be the owner? These constants are, in practice, part of the public interface of the services, they are needed to any client that want to send and Intent to the service, they are part of the contract. And the contract should be defined by the server part, so that different clients can access to the server.